core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/ColumnFormatRelation.scala

```diff
@@ -323,19 +323,14 @@ abstract class BaseColumnFormatRelation(
       if (tableExists) {
         mode match {
           case SaveMode.Ignore =>
-            // TODO: Suranjan for split mode when driver acts as client
-            // we will need to change this code and add a flag to
-            // CREATE_ALL_BUCKETS to create only when no buckets created
-//            if (region.getRegionAdvisor().getCreatedBucketsCount == 0) {
```

Comment:
Didn't understand this comment and corresponding code on store side. What's the need to skip CREATE_ALL_BUCKETS? Also that check should not be inside CREATE_ALL_BUCKETS since that API says to create any missing buckets, rather a new one added to get this behaviour if required.